tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
As a side note in reply to "part of the same project", there is nothing in the load balancer role that is specific to the instance manager.  It can be useful wherever you want an LB that manages SSL certificates for you.